example/requirements.txt

```diff
@@ -1,2 +1,2 @@
-django
-https://github.com/juanifioren/django-oidc-provider/archive/master.zip
+django==3.2.18
+../
```

Comment:
pin django version at what chassis is currently using
used relative path so that the example app would test the packages code not the repo.